screens/AnnouncementScreen.js

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
-import { ScrollView, Text, TouchableOpacity, View } from 'react-native';
+import {AsyncStorage, ScrollView, Text, TouchableOpacity, View} from 'react-native';
 
 import Announcements from '../components/Announcements';
 import BASE from '../lib/common';
-
+import * as Font from "expo-font";
+import {
+  TopText,
+} from '../styles/announcements';
 const announceTable = BASE('Announcements').select({ view: 'Grid view' });
 const firstAnnouncements = [];
+
+Font.loadAsync({
+  Poppins: require('../assets/fonts/Poppins-Regular.ttf')
+});
+
 announceTable.eachPage(
```

Comment:
I'd recommend doing this inside `componentDidMount` instead. I don't remember the exact details but I don't believe it's good practice to have important code running here -- we want to keep it within the component lifecycle